src/tools/ceph_objectstore_tool.cc

```diff
@@ -2601,6 +2660,16 @@ int main(int argc, char **argv)
     goto out;
   }
 
+  if (object == "apply-layout-settings") {
```

Comment:
I originally developed this against hammer which had another command as a positional arg already. Using --op apply-layout-settings sounds better, I'll switch to that. At that point would it make more sense to make the positional object arg optional, and just use --pgid or --pool for this command?